app/javascript/flavours/glitch/styles/components/local_settings.scss

```diff
@@ -110,6 +110,10 @@
       text-decoration: none;
     }
   }
+
+  #mastodon-settings--collapsed-auto-height {
+    width: 45px
+  }
```

Comment:
Why a specific style for this?

Reply:
This field looked ridiculous without having its width restricted, and InputProps does not support the width parameter for all browsers

It's there another way you'd prefer it done?

Reply:
Oh, I see! A hardcoded pixel width doesn't sound safe though wrt. potentially different fonts and browser layout. I would suggest to use `em` or `ch` but that is not quite right either as the size needs to include the up/down arrows, which is browser-dependent… `calc(3ch + 20px)` is equivalent to your proposal but should scale a bit depending on the used font… but it's no guarantee either. Also we may want to allow space for 4 characters, so `calc(4ch + 20px)`.